README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+# hw01
+## Autobiography
+My name is Olivia Padilla, and I am a third-year undergraduate studying Public Policy.  I am from a very small town in the rural Ozarks region of southwest Missouri. Here is a link to my town's webpsite. [link](http://www.cityofcassville.com/). In my free time I like to read, my favorite book is *The Phantom Tollbooth* by Norton Juster. I have three goals for this quarter: 
+1. Practice social distancing. 
+2. Maintain my mental health while doing so. 
+3. Plan an awesome cross-country road-trip for when this is all over.  
+
+## Process
+1. I watched the video Professor Soltoff posted for this assignment. I made notes so that I could follow the correct process by myself.
+2. I forked Homework 01's repository from the class GitHub to my own repository.
+  + I cloned the fork to my computer and copied the repository's URL.
+3. I opened a new project in RStudio Server.
+  + I created a "Version Control Project" with Git.
+  + I used the repository's URL to create this project. 
```

Comment:
This is a good example of how simple stylistic changes can helps with better interpretation/visualization. Rather than a long paragraph, the way you ordered and indented your lists communicates a more sequential pattern of steps, almost like following a recipe. It is easier to understand this way. This is something that I should have done!